media-libs/libopenshot-audio/libopenshot-audio-0.0.6.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Audio library used by OpenShot"
+HOMEPAGE="http://www.openshot.org/ https://launchpad.net/libopenshot"
+SRC_URI="https://launchpad.net/libopenshot/0.1/0.1.0/+download/${P}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
```

Comment:
It's common (but not required) to sort these as ```LICENSE / SLOT / KEYWORDS / IUSE``` - just an FYI in case this wasn't intentional.